.circleci/config.yml

```diff
@@ -281,14 +279,11 @@ commands:
       - run:
           name: Linter check
           command: |
-            ng lint
-
+            ng lint ddp-sdk
+            ng test toolkit
+            ng test $ANGULAR_PROJECT_NAME
   run-tests:
```

Comment:
Is `ng test` correct for lines 283-284 ?  
This piece/step is called "Linter check".